### ISSUE TITLE
Training with pascal VOC goes wrong 

### ISSUE THREAD

**Issue by SanthoshRajendiran** [gallifilo/final-year-project#2]
Hello.. We tried your training script with portrait images, the training goes fine.. But with pascal VOC, since the masks are different, we uncommented part of the code in loader.py

        # Change indices which correspond to "void" from 255
        images_segmented = np.where((images_segmented != 15) & (images_segmented != 255), 0, images_segmented)
        #images_segmented = np.where(images_segmented == 15, 1, images_segmented)
        #images_segmented = np.where(images_segmented == 255, len(DataSet.CATEGORY)-1, images_segmented)
        images_segmented = np.where(images_segmented == 255, 1, images_segmented)
        #images_segmented = np.where(images_segmented == 254, len(DataSet.CATEGORY)-1, images_segmented)

All these commented lines, we uncommented, as the masks has to be handled differently for Pascal VOC.. Still the output goes bad, even around 22k epochs.. Is this methodology correct or what else needs to be done? And at how many epochs can we expect a decent enough output?
